src/doc/style-guide/src/items.md

```diff
@@ -450,7 +450,7 @@ entries, format it across multiple lines as with a type alias.
 
 When writing extern items (such as `extern "C" fn`), always specify the ABI.
 For example, write `extern "C" fn foo ...`, not `extern fn foo ...`, or
-`extern "C" { ... }`.
+`unsafe extern "C" { ... }`.
```

Comment:
The style guide only describes stable formatting. 

Please add this to the nightly formatting chapter per the nightly style policy: https://doc.rust-lang.org/nightly/style-guide/nightly.html

Reply:
Unless you want to block this until stabilization is imminent, in which case you probably should write something in the description to clarify what your intention is here. Right now you've provided basically no information about this.

Reply:
Sorry, I did it in the other PRs I've opened but didn't in this one. Also the intention was to link from the tracking issue so it's visible in github that there's a connection between the issues but when I was going to do I was distracted with other stuff and couldn't complete it.